tests/testsuite/doc.rs

```diff
@@ -1168,6 +1168,39 @@ fn doc_workspace_open_help_message() {
         .run();
 }
 
+#[cargo_test]
+#[cfg(not(windows))] // `echo` may not be available
+fn doc_extern_map_local() {
+    assert!(is_nightly());
```

Comment:
This assert will cause the test to fail on stable.  To make the test conditional on being nightly, it should be something like the following:

```suggestion
    if !is_nightly() {
        // --extern-html-root-url is unstable
        return;
    }
```